blazor/scheduler/dimensions.md

```diff
@@ -112,4 +112,37 @@ When height and width of the Scheduler are given as percentage, it will make the
         public Nullable<int> RecurrenceID { get; set; }
     }
 }
+```
+
+## How to set the full height of the view port
+
+When height of the scheduler is set as "calc(100vh - 4.6rem)", it will make the scheduler to occupy the full height within the view port.
```

Comment:
When the height of the scheduler is set as "calc(100vh - 4.6rem)", it will make the scheduler to occupy the full height within the view port.